### PR TITLE
Adding new problem

### PR DESCRIPTION
**Pull request by saraisabellabc**
For assignment 3 of SF-1

### CODE REVIEW

**Comment by Mahaefma** [comment]
-Nice clean JSON formatting! No missing commas or brackets to fix 👍
-Great examples in the description - makes it super clear what the function should do
-Smart test case with 'EricMayhewIsTHEbest' catching that uppercase/lowercase issue!
-Really like how you laid out the function header with proper type hints in starter.py
-You could add a test case for empty strings just to make sure it handles that case correctly

File: middle-characters/io.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "input": "rawAwArawrrr",
```
Comment (emaj9) [inline middle-characters/io.json:3]: inputs need to be in square brackets
